docs/installation/packages.md

```
@@ -0,0 +1,42 @@
+# Package Installation (deb / rpm)
+
+Prerelease packages are published on the [Releases](https://github.com/yorishiro-ai/yorishiro/releases) page alongside each Docker image.
+
+They support Ubuntu 24.04 and AlmaLinux 10 (glibc 2.39+).
+
+## Install
+
+```console
+$ wget https://github.com/yorishiro-ai/yorishiro/releases/download/20260906-1/yorishiro_20260906-1_amd64.deb
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)